detekt.yml

```diff
@@ -270,6 +270,9 @@ potential-bugs:
 
 style:
   active: true
+#  TODO This should possibly be enabled; it should be discussed
```

Comment:
❓ I got a ton of failures for this check that we hadn't been getting before. I don't know why it changed.

We need to decide if we want to enforce braces on one-line ifs or not. It feels a little un-kotliny, but it is a tighter match with our Java standard so there are arguments both ways.